es-app/src/SystemScreenSaver.cpp

```diff
@@ -314,7 +314,7 @@ void SystemScreenSaver::backgroundIndexing()
 	std::vector<FileData*> files = all->getRootFolder()->getFilesRecursive(GAME);
 
 	const auto startTs = std::chrono::system_clock::now();
-	for (lastIndex; lastIndex < files.size(); lastIndex++)
```

Comment:
Compiler warning:

/home/devel/temp/EmulationStation_Stable/es-app/src/SystemScreenSaver.cpp:317:7:
 warning: expression result unused [-Wunused-value]
        for (lastIndex; lastIndex < files.size(); lastIndex++)
             ^~~~~~~~~